app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+
+  def register
+    # @user = User.new
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save
```

Comment:
we might want to manually check params if save() isnt returning false/something is going wrong. it might be interpreting the black field as an empty string, which is truthy i think